modules/combined/tests/contact_verification/patch_tests/cyl_3/cyl3_template1.i

```diff
@@ -291,9 +291,6 @@
   #Preconditioned JFNK (default)
   solve_type = 'PJFNK'
 
-  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
-  petsc_options_value = 'lu     superlu_dist'
-
```

Comment:
What's the point in removing this from the input file and putting it cli args for every single test?  I'd prefer to leave these as-is if we can.